packages/main-ui/src/pages/components/loader_popup.rs

```diff
@@ -10,41 +13,62 @@ pub fn LoaderPopup(
     #[props(default ="loader_popup".to_string())] id: String,
     #[props(default ="".to_string())] class: String,
     lang: Language,
+    title: String,
+    description: String,
     logo: Element,
     logo_origin: Element,
     msg: String,
 ) -> Element {
-    let tr = translate::<LoaderPopupTranslate>(&lang);
     let mut user_service: UserService = use_context();
     let mut popup: PopupService = use_context();
-
+    let display_logo = logo.clone();
+    let load_message = description.clone();
     use_effect(move || {
-        let logo = logo_origin.clone();
+        let logo_origin = logo_origin.clone();
+        let logo = logo.clone();
         let msg = msg.clone();
+        let description = description.clone();
         spawn(async move {
             match user_service.login().await {
                 UserEvent::Signup(principal, email, nickname, profile_url) => {
-                    popup.open(rsx! {
-                        UserSetupPopup {
-                            class: "w-[400px] mx-[5px]",
-                            nickname,
-                            profile_url,
-                            email,
-                            principal,
-                            lang,
-                        }
-                    });
+                    popup
+                        .open(rsx! {
+                            UserSetupPopup {
+                                class: "w-[390px]",
+                                nickname,
+                                profile_url,
+                                email,
+                                principal,
+                                lang,
+                            }
+                        })
+                        .with_id("user_setup_popup");
                 }
                 UserEvent::Login => {
                     popup.close();
                 }
+                UserEvent::Confirmed => {
+                    tracing::info!("User confirmed");
```

Comment:
don't use info for observing behaviour. info means something users should recognize